site/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "astro/tsconfigs/strict",
+    "include": [".astro/types.d.ts", "**/*"],
+    "exclude": ["dist"],
+    "compilerOptions": {
+        "baseUrl": "./",
+        "jsx": "react-jsx",
+        "jsxImportSource": "react"
+    }
```

Comment:
I tried to make some issues go away  by adding lib: dom and other bundler settings, but something about this is just not working for me - doing so resolves my missing-all-imports issue, but at the cost of making Typescript un-resolvable, so basic stuff like "Record" and "Promise" become errors

Reply:
maybe the examples could be imported by site? I think I remember you said you tried that and it didnt work out right?

Reply:
Hm. Yeah, I could try it again having them separate. It just complicates the whole process 🤔